.github/CODEOWNERS

```diff
@@ -25,6 +25,6 @@
 /doc/values.md @NixOS/steering
 /doc/vault.md @Mic92 @mweinelt @zimbatm
 /doc/constitution.md @NixOS/steering
-/doc/nixpkgs-committers.md @Mic92 @NickCao @jtojnar
+/doc/nixpkgs-committers.md @Mic92 @NickCao @jtojnar @winterqt
```

Comment:
Conflicts with #108, will rebase after.

Reply:
```suggestion
/doc/nixpkgs-committers.md @NickCao @jtojnar @winterqt
```

Reply:
Fair, we can do it here.

Reply:
(though note I left it there because @infinisil changed it in #108)